molo/core/tests/test_views.py

```diff
@@ -141,6 +141,10 @@ def test_able_to_copy_main(self):
             title='banner', slug='banner', banner_link_page=article)
         self.banner_index.add_child(instance=banner)
         banner.save_revision().publish()
+        banner2 = BannerPage(
+            title='banner2', slug='banner2', banner_link_page=self.yourmind)
+        self.banner_index.add_child(instance=banner2)
+        banner2.save_revision().publish()
```

Comment:
@nathanbegbie agree, we should add more specific tests for `create_new_article_relations` function in `test_utils` but in this PR I'm mostly aiming to fix the current production bug. will create an issue for us to add more specific tests